mastering-plone/viewlets_1.md

```diff
@@ -135,8 +135,18 @@ class FeaturedViewlet(ViewletBase):
 ```
 
 
+
+So far, we
+
+> - register the viewlet to content that has the IFeatured Interface.
+> - adapt the object to its behavior to be able to access the fields of the behavior
+> - return the link
```

Comment:
Do we need `>`? It puts the list into a quote block, and I think that is unnecessary.